test/elm/interval/test.coffee

```diff
@@ -2124,3 +2124,111 @@ describe 'DecimalIntervalExpand', ->
     # define BadPerMinute: expand { Interval(2.1, 4.1] } per 0.5 minute
     a = @badPerMinute.exec(@ctx)
     should.not.exist(a)
+
+describe 'SameAs', ->
+  @beforeEach ->
+    setup @, data
+
+  it 'returns null when both intervals values are null', ->
+    # define NullBoth: Interval[null,null] same as Interval[null,null]
```

Comment:
While I can see the benefit of looking at these while writing the tests, I would remove these comments showing the cql from this file, as if the CQL changes for some reason, these will become stale.

Reply:
Decided to keep these because that is the format for the majority of the file